src/device/usbd.c

```diff
@@ -765,9 +765,7 @@ static bool process_set_config(uint8_t rhport, uint8_t cfg_num)
         {
           // IAD's first interface number and class/subclass/protocol should match with opened interface
           TU_ASSERT(desc_itf_assoc->bFirstInterface   == desc_itf->bInterfaceNumber   &&
```

Comment:
@PanRe Kasjer is mentionening about the white space before the `==` it is there since `bFunctionSubClass & bFunctionProtocol` is too long, since it is removed, we could just remove a few spaces and still keep the alignment.